policies/rules.md

```diff
@@ -3,108 +3,115 @@ layout: default
 title: CODEVID-19 Rules
 ---
 
-## Summary
+# Quick Links
```

Comment:
I like the idea of highlighting these related items, but `FAQ` and `Submission` need to stay in the top menu.

Reply:
Is this needed anymore if we keep Submission and FAQ in the nav?

Reply:
I'll just keep it in for now.

Reply:
I'd be fine with taking the quick links out, they're easy to add back in if needed.